src/modules/MouseWithoutBorders/App/Class/Common.Helper.cs

```diff
@@ -327,7 +325,7 @@ internal static void RunDDHelper(bool cleanUp = false)
             }
             else
             {
-                if (processes != null && processes.Length != 0)
+                if (Process.GetProcessesByName(HelperProcessName).Length > 0)
```

Comment:
The original code null checked the return by "?". Why are we removing that? If there's a possibility that there's a null return, we'll crash with the current code.